src/lib/validate-forms.js

```diff
@@ -0,0 +1,67 @@
+const api = require('./api')();
+const argsFormFilter = require('./args-form-filter');
+const fs = require('./sync-fs');
+const log = require('./log');
+const { getFormDir, getFormFilePaths } = require('./forms-utils');
+
+//TODO Add this to the main loop
```

Comment:
What do you mean by main loop? Can this TODO be fixed, or removed, or converted into an issue?

Reply:
Yes it is an old TODO that I should remove before pushing